### DISCUSSION TITLE
Problem when using Loihi2HwCfg()

### DISCUSSION THREAD

**Discussion by Andy-Urbano** [lava-nc/lava#512] [General]
I have encontered a problem when trying to run a recurrent SNN on Loihi2. I have access to the INRC servers, but when running the code, I receive an error saying there is a missing CLoihiProcessModel undefined.
From what I have read in this repository doesn't make it seem like others are running into the same problem, so I might be making some basic mistake. Would appreciate any help.

**Comment by weidel-p** [comment]
Could you try to create a minimal example which shows this behavior and share that here? It's difficult to help without actually seeing what you are trying to do.

**Comment by CloudyDory** [comment]
@Andy-Urbano You should check the tutorials provided in file `lava-loihi-0.6.1.tar.gz` in the INRC servers. The `tutorial00_tour_through_lava.ipynb` has an extra session at the end (compared to the tutorial provided in this repository) that says you how should write the neurons in C code in order to let it run on Loihi hardware.
